React-Todo-list-main/src/App.js

```diff
@@ -19,25 +15,58 @@ function App() {
   /**
    * initialzing user
    */
-  const user = {
-    userID: "user-us",
-    email: "upendra.singh@ltts.com",
-  };
+
+  const [user, setUser] = useState(JSON.parse(localStorage.getItem("user")));
+
   console.log("User: ", user);
 
+  const [isLoggedIn, setIsLoggedIn] = useState(
+    localStorage.getItem("isLoggedIn") === "true"
+  );
+
+  /**
+   * Setting the user and flag from the login details
+   * @param {*} userName
+   */
+  const setUserDetails = (userName) => {
+    let localUser = {
+      userID: userName,
+      email: "upendra.singh@ltts.com",
```

Comment:
make a random email as well - username@todoapp.com

Reply:
Added this dummy email id